schemas/dab.draft.schema.json

```diff
@@ -177,7 +177,7 @@
               "description": "Allow enabling/disabling GraphQL requests for all entities."
             },
             "depth-limit": {
-              "type": "integer",
+              "type": [ "integer", "null" ],
```

Comment:
we have to explicitly state that a field is nullable by include type `null` ? wouldn't this be required throughout this entire file? did you encounter `dab validate` errors?

Reply:
yes, schema validation was complaining that `depth-limit` is of type integer when providing `null` value.

Reply:
@abhishekkumams what's the error printed? Why do we need to add `null` for only this field and not all others in the config? is there something different about this property? we have caching `ttl-seconds` which doesn't seem to result in error when set to null. Is this another bug that needs to be fixed?

Reply:
i didn't check `ttl-seconds` are added in validation tests. But schema validation should fail with the value set as `null` if not specified in `dab.draft.schema.json`

Reply:
created #2277